sdk/src/utils/test.rs

```diff
@@ -371,6 +409,102 @@ impl crate::signer::RemoteSigner for TempRemoteSigner {
     }
 }
 
+#[cfg(target_arch = "wasm32")]
```

Comment:
Is this signer meant to be real or stay as a test?  If it is real can you move it to the wasm folder

Reply:
The implementation can be moved to wasm folder. But I just had the logic there for a ES256 certificate.  Eventually we will have to add logic  for other certificate types (or fail).  Also, it is only used in test code now, not sure if we want to expose this for taget_arch="wasm32"

Reply:
Ok, leave this as a test signer until we figure this out.